cfme/tests/v2v/test_migration_throttling.py

```diff
@@ -0,0 +1,93 @@
+import fauxfactory
+import pytest
+from widgetastic.widget import NoSuchElementException
+
+from cfme import test_requirements
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.fixtures.provider import rhel69_template
+from cfme.fixtures.provider import rhel7_minimal
+from cfme.fixtures.provider import ubuntu16_template
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.markers.env_markers.provider import ONE_PER_TYPE
+from cfme.markers.env_markers.provider import ONE_PER_VERSION
+from cfme.v2v.migration_settings import MigrationSettings
+
+pytestmark = [
+    test_requirements.v2v,
+    pytest.mark.provider(
+        classes=[OpenStackProvider, RHEVMProvider],
+        selector=ONE_PER_VERSION,
+        required_flags=["v2v"],
+        scope="module"
+    ),
+    pytest.mark.provider(
+        classes=[VMwareProvider],
+        selector=ONE_PER_TYPE,
+        fixture_name="source_provider",
+        required_flags=["v2v"],
+        scope="module",
+    ),
+    pytest.mark.usefixtures("v2v_provider_setup"),
+]
+
+
+@pytest.mark.parametrize(
+    "mapping_data_multiple_vm_obj_single_datastore",
+    [
+        ["nfs", "nfs", [rhel7_minimal, rhel69_template, ubuntu16_template]],
```

Comment:
Q: Why parametrize against a single parameter? Plans to parametrize against more?

Reply:
The fixtures like mapping_data_multiple_vm_obj_single_datastore in v2v_fixtures.py are written to take parameters from requests hence paramterized this test.

https://github.com/ManageIQ/integration_tests/blob/master/cfme/fixtures/v2v_fixtures.py#L467